ocs_ci/ocs/ui/views.py

```diff
@@ -628,7 +647,7 @@
         By.XPATH,
     ),
     "object_storage": ("//a[normalize-space()='Object Storage']", By.XPATH),
-    "observe": ("//button[text()='Observe']", By.XPATH),
+    "Monitoring": ("//button[text()='Monitoring']", By.XPATH),
```

Comment:
Monitoring need to be changed back to Observe; this is one of the recent changes probably accidentally pulled from the stale master branch

Please do rebase to master

Reply:
Sorry, fixed